src/control/cmd/dmg/main.go

```diff
@@ -82,37 +79,89 @@ func appSetup(broadcast bool) error {
 	return nil
 }
 
-func main() {
-	if dmgMain() != nil {
-		os.Exit(1)
-	}
+type broadcastCmd struct{}
```

Comment:
could we have a description for broadcast and connected Cmd types as it's not immediately clear.

Reply:
Sure, I'm going to move all of these declarations up into a single type block as well.